lib/layouts/root_layout.ex

```diff
@@ -8,10 +8,14 @@ defmodule TableauDemoHeex.RootLayout do
     <html>
       <head>
         <title>Tableau EEx Demo</title>
+        <link rel="stylesheet" type="text/css" href="/css/site.css" />
       </head>
       <body>
         <%= render(@inner_content) %>
       </body>
+      <%= if Mix.env() == :dev do %>
+        <%= {:safe, Tableau.live_reload(assigns)} %>
```

Comment:
Pretty sure this should be `<Tableau.live_reload />`

Reply:
Unfortunately, not for me. I get:

```shell
** (RuntimeError) expected &Tableau.live_reload/1 to return a %Phoenix.LiveView.Rendered{} struct

Ensure your render function uses ~H to define its template.
```

It currently returns only a string instead of a Heex-Component. Or am i missing anything?

Reply:
oh, actually I think that makes sense. under the hood its just an EEx.compile_function, so its a string.

Okay, in this case, you should do basically the same thing, but it'll be `<%= Phoenix.HTML.raw(Tableau.live_reload(assigns) %>`, or, can make a component elsewhere in this demo 

```elixir
def live_reload(assigns) do
  ~H"""
  <%= Phoenix.HTML.raw(Tableau.live_reload(assigns) %>
  """
end
```

```
<.live_reload />
```

Reply:
Thanks for the Feedback! I've implemented a function component like you suggested 😉